pom.xml

```diff
@@ -36,6 +37,12 @@
       <url>https://www.gnu.org/licenses/old-licenses/gpl-2.0.txt</url>
     </license>
   </licenses>
+
+  <modules>
+    <module>src/shared</module>
+    <module>src/modules</module>
+    <module>src/apps</module>
```

Comment:
Not sure what value the `src` is having here?  Is it a convention for spring-boot multi-module apps?

- Q: What is "shared"? Is it common code shared between several plugins?
- Q: What is "modules"? Everything in maven is a module; perhaps it is something else ...
-Q: To confirm plural "apps" and "modules"?


Recommend `shared`, `module`, `app`.

Reply:
No specific convention I think. @josegar74 ?

https://github.com/geonetwork/geonetwork-microservices/ was using another layout.

> Q: What is "shared"? Is it common code shared between several plugins?

Yes

> Q: What is "modules"? Everything in maven is a module; perhaps it is something else ...

We started naming it "extension" but we were not convinced :)
Modules will contain each components/features in the application ... 

> Q: To confirm plural "apps" and "modules"?

No strong opinion.